src/app/public/modules/description-list/description-list.component.html

```diff
@@ -0,0 +1,27 @@
+<dl
+  class="sky-description-list"
+  [ngClass]="{
+      'sky-description-list-long-description-mode' : mode == 'longDescription',
+      'sky-description-list-vertical-mode' : mode == 'vertical',
+      'sky-description-list-horizontal-mode' : mode == 'horizontal'
+    }"
+  #descriptionListElement
+>
+  <!--
+    Wrapping div is for horizontal flexing. Also injecting children's templateRef directly
+    to avoid adding non-semantic HTML tags to DOM
+  -->
+  <div *ngFor="let contentComponent of contentComponents"
+    class="sky-description-list-content"
+    [style.width]="mode === 'horizontal' ? listItemWidth : ''"
+  >
+    <dt>
+      <ng-container *ngTemplateOutlet="contentComponent?.termComponent?.templateRef"></ng-container>
+    </dt>
+    <dd>
+      <ng-container *ngFor="let descriptionComponent of contentComponent?.descriptionComponents;">
```

Comment:
Shouldn't the `*ngFor` be on the `dd` element?

Also, it appears that description lists can contain multiple `dt`s for a single `dd`. Is that something we need to support? I'm guessing not. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl#multiple_terms_single_description

Reply:
Yep it should. Thanks for catching that! I've added the loop to both term and description.